core/controllers/cron.py

```diff
@@ -223,3 +227,28 @@ def get(self):
             cron_services.JobModelsCleanupManager.enqueue(
                 cron_services.JobModelsCleanupManager.create_new())
             logging.warning('Deletion jobs for JobModels entities kicked off.')
+
+
+class CronMailContributorDashboardReviewerOpportunitiesHandler(
+        base.BaseHandler):
+    """Handler for mailing reviewers review opportunities on the Contributor
+    Dashboard.
+    """
+
+    @acl_decorators.can_perform_cron_tasks
+    def get(self):
+        """Sends each reviewer an email with up to
+        MAX_NUMBER_OF_SUGGESTIONS_TO_EMAIL_REVIEWER suggestions that have been
```

Comment:
Where is this constant defined? Please qualify it so that readers know where to look for it.